raiden/ui/console.py

```diff
@@ -130,7 +130,7 @@ def lasterr(n=1):
         self.console_locals = {
             "app": self.app,
             "raiden": self.app.raiden,
-            "chain": self.app.raiden.chain,
+            "chain": self.app.raiden.proxy_manager,
```

Comment:
shouldn't we also change the configuration name? Instead of chain put "proxy_manager" ?

Reply:
Nice catch